using-timescaledb/continuous-aggregates.md

```diff
@@ -151,6 +151,13 @@ will be spawned immediately following completion. This means that the part of th
 data processed by previous jobs will be available even before the
 materialization is fully caught up.
 
+The `timescaledb.ignore_invalidation_older_than` parameter control how
+adding records with an old timestamp will trigger update of the
+continuous aggregate. An update that is older than this parameter will
+not trigger an update so, for example, setting the parameter to "1
+month" will ensure that any update for a record that is older than 1
+month will not trigger an update of the view.
+
```

Comment:
we should also add a note here about how this affects data retention. Is that documented anywhere else? (maybe a link to drop_chunks_policy or material from a  blog).

Reply:
ok, lets do this when we add the section to data-retention.md. Need an example on how to do data retention for continuous aggregates as part of that section.

Reply:
I will add a section as a separate PR.

Reply:
It is available in https://github.com/timescale/docs.timescale.com-content/pull/314